src/applications/vaos/services/referral/index.js

```diff
@@ -0,0 +1,29 @@
+import { apiRequestWithUrl } from '../utils';
+
+const acheronHeader = {
+  headers: { ACHERON_REQUESTS: 'true' },
+};
+
+export async function getPatientDetails(
```

Comment:
I think what we really need for this PR is to fetch a single referral by id. We can rename this to getReferral (to fetch a single referral by id).  
My guess is we should only need referralID as a param, the ID will be gotten from the previous page.